tests/by-util/test_hashsum.rs

```diff
@@ -356,6 +356,12 @@ fn test_invalid_arg() {
     new_ucmd!().arg("--definitely-invalid").fails().code_is(1);
 }
 
+#[test]
+fn test_conflicting_arg() {
+    new_ucmd!().arg("-tag").arg("--check").fails().code_is(1);
+    new_ucmd!().arg("-tag").arg("--text").fails().code_is(1);
+}
```

Comment:
This test passes even without the changes made to `hashsum.rs` because it only tests for a failure. Currently the test passes because `-tag` causes an `unexpected argument` failure. And even with this fixed, another failure ("no hash algorithm specified") makes the test still pass. So I would either specify an algorithm or verify the correct error message is shown. 
```suggestion
fn test_conflicting_arg() {
    new_ucmd!()
        .arg("--tag")
        .arg("--check")
        .arg("--md5")
        .fails()
        .code_is(1);
    new_ucmd!()
        .arg("--tag")
        .arg("--text")
        .arg("--md5")
        .fails()
        .code_is(1);
}
```